### DISCUSSION TITLE
Navigating GitHub as a New User: Need Help!

### DISCUSSION THREAD

**Discussion by solarwalker** [community/community#136007] [Discussions] [answered]
### Select Topic Area

Question

### Hello User,
I’m new to the platform and trying to get a handle on how to use GitHub effectively. There’s so much to learn, and I’d appreciate any advice on where to begin, how to find good projects to contribute to, and how to manage my own repositories. Any tips from experienced users would be fantastic!

**Comment by Deepayan-Thakur** [comment, marked as answer]
You must Add some content related to DSA Prep as it must be give you some ideas about how to be relevant in todays world. And by learning new data structures as it must be having some of the core concepts for creating software.
